src/Stripe.net/Entities/Payouts/Payout.cs

```diff
@@ -85,21 +65,11 @@ internal object InternalDestination
         public string FailureBalanceTransactionId { get; set; }
 
         [JsonIgnore]
-        public BalanceTransaction FailureBalanceTransaction { get; set; }
+        public BalanceTransaction FailureBalanceTransaction => this.InternalFailureBalanceTransaction.ExpandedObject;
```

Comment:
OOC, should we add a `FailureBalanceTransactionId` for this one?

Reply:
Oops, nice catch! Looks like my codemod missed this one.